eZ/Publish/API/Repository/ContentService.php

```diff
@@ -148,6 +148,20 @@ public function loadContent($contentId, array $languages = null, $versionNo = nu
      */
     public function loadContentByRemoteId($remoteId, array $languages = null, $versionNo = null, $useAlwaysAvailable = true);
 
+    /**
+     * Bulk-load Content items by the list of ContentInfo Value Objects.
+     *
+     * Note: it does not throw exceptions on load, just ignores erroneous Content item.
+     *
+     * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo[] $contentInfoList
+     * @param string[] $languages A language priority, filters returned fields and is used as prioritized language code on
+     *                            returned value object. If not given all languages are returned.
+     * @param bool $useAlwaysAvailable Add Main language to \$languages if true (default) and if alwaysAvailable is true
```

Comment:
Unless all languages have been asked for

Reply:
Fixed doc via fixup 3093853